plugins/available/blesh.plugin.bash

```diff
@@ -0,0 +1,9 @@
+# shellcheck shell=bash
+cite about-plugin
+about-plugin 'load ble.sh, the Bash line editor!'
+
+if [[ -f "$HOME/.local/share/blesh/ble.sh" ]]; then
+	source "$HOME/.local/share/blesh/ble.sh"
+else
+	_log_error "Could not locate ble.sh, please follow installation instructions from https://github.com/akinomyoga/ble.sh"
```

Comment:
This is probably a good idea

Reply:
Or maybe we can explicitly display the commands to install `ble.sh`. I'd think it would be more user-friendly.

Reply:
The problem is that the instructions could change in the future. I feel more comfortable referring users to your repo instead @akinomyoga

Reply:
I think the installation instruction is unlikely to be changed ever. Even if I would add a new way of installation, I think I would try to keep the current way working for backward compatibility. Also, in case it would be changed, I will submit a PR to `bash-it`. Actually, the current instruction in `README.md` is already somewhat a compatibile form. Now a simple `git clone` & `make install` should work for this `blesh.plugin.bash`:

```bash
$ git clone https://github.com/akinomyoga/ble.sh
$ make -C ble.sh install
```

Another confusing point of just telling "*follow the instruction in README*" is that that `README.md` tells users to add the line `source ble.sh` in their `.bashrc`, but that doesn't apply to the case of `blesh.plugin.bash`. I assume that users don't know `blesh.plugin.bash` actually does `source ble.sh` in prior, so I wouldn't be surprised if a user sources `ble.sh` in the user's `~/.bashrc` and also enables `blesh.plugin.bash`.

Reply:
[Resolve conversation]